http/sys_mount_test.go

```diff
@@ -1829,6 +1832,114 @@ func TestSysTuneMount_passthroughRequestHeaders(t *testing.T) {
 	}
 }
 
+func TestSysTuneMount_identityTokenKey(t *testing.T) {
```

Comment:
I followed this pattern after looking at other tune tests in this file. After having written tests in `logical_system_test.go`, I'm thinking it might be better to move the tune test there. Going to give it a shot and see.

Reply:
Moved to `logical_system_test.go` in https://github.com/hashicorp/vault/pull/24962/commits/9e66566e8a397adf1116dde47288d9c1c28ff612.